src/TensorSharp/Core/MemoryCopier.cs

```diff
@@ -16,8 +16,8 @@ private static class NativeMethods32
 
         private static class NativeMethods64
```

Comment:
Since we are moving to .NET core, we should strive to make this portable. I suggest that we delete this whole file, and replace calls to `MemoryCopier.Copy` with calls to `System.Buffer.MemoryCopy` instead. This gives us way to achieve the same thing using a .NET Standard compatible API. A quick search suggests there are only 2 or 3 call sites to update

Reply:
OK, will do